### PR TITLE
Refactor. Core build mode.

### PR DESCRIPTION
**Pull request by blckmn**

### CODE REVIEW

File: locales/en/messages.json
Comment (haslinghuis) [inline locales/en/messages.json:6855]: This is likely to change soon, so better remove:

> for example GPS and LED_STRIP are not available on smaller flash sized targets like F722 and F411 using this mode.

**Comment by McGiverGim** [comment]
Are this rename necessary at this stage? Again, it produces that all the translators need to translate it again.

**Comment by blckmn** [comment]
> Are this rename necessary at this stage? Again, it produces that all the translators need to translate it again.

It's only renamed in english, the existing translations can remain - and if they get updated great, if not there is no harm done as "classic" is still a valid description.

File: locales/en/messages.json
Comment (haslinghuis) [inline locales/en/messages.json:6852]: Core Mode ?

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

File: locales/en/messages.json
Comment (blckmn) [reply locales/en/messages.json:6852]: I am not sure it is a "mode". If anything it would be `Core Only`

Comment (haslinghuis) [reply locales/en/messages.json:6852]: Please abandon term classic

**Review by haslinghuis — APPROVED** [review]

**Comment by McGiverGim** [comment]
> > Are this rename necessary at this stage? Again, it produces that all the translators need to translate it again.
> 
> It's only renamed in english, the existing translations can remain - and if they get updated great, if not there is no harm done as "classic" is still a valid description.

That does not work in this way. When you change the English translation, at Crowdin, all the translations are invalidated. Your work here modifying all the languages does not serve for nothing because it will be replaced with Crowdin files the next update.

**Review by chmelevskij — APPROVED** [review]

**Pull request merged**
